src/About/About.module.css

```diff
@@ -45,3 +72,33 @@ ol {
 .wrap {
   text-align: center;
 }
+
+.item {
+  position: relative;
+}
+
+.update {
+  font-size: 10px;
+  position: absolute;
+  right: 10px;
+}
+
+@media (max-width: 430px) {
+  .bio > img {
+    width: 300px;
+    height: 300px;
+  }
+
+  .link {
+    width: 65%;
+    font-size: 13px;
+  }
+
+  .update {
+    font-size: 8px;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/dotodo/pullRequest?prid=6099781)